tools/tx_pull.py

```diff
@@ -0,0 +1,222 @@
+                    tx_lang = False
+                    cont += 1
+            if tx_lang:
+                try:
+                    tx_po_file = polib.pofile(tx_lang['content'])
+                    tx_po_dict = _load_po_dict(tx_po_file)
+                    # Discard empty languages
+                    if not tx_po_dict:
+                        continue
+                    gh_i18n_path = os.path.join('/', resource['slug'], "i18n")
+                    gh_file_path = os.path.join(
+                        gh_i18n_path, lang['code'] + '.po')
+                    gh_file = gh_repo.contents(gh_file_path, gh_branch.name)
+                    if gh_file:
+                        gh_po_file = polib.pofile(
+                            gh_file.decoded.decode('utf-8'))
+                        gh_po_dict = _load_po_dict(gh_po_file)
+                        unmatched_items = (set(gh_po_dict.items()) ^
+                                           set(tx_po_dict.items()))
+                        if not unmatched_items:
+                            print "...no change in %s" % gh_file_path
+                            continue
+                    print '..replacing %s' % gh_file_path
+                    new_file_blob = gh_repo.create_blob(
+                        tx_lang['content'], encoding='utf-8')
+                    tree_data.append({
+                        'path': gh_file_path[1:],
+                        'mode': '100644',
+                        'type': 'blob',
+                        'sha': new_file_blob})
+                except (KeyboardInterrupt, SystemExit):
+                    raise
+                except:
+                    print "ERROR: processing lang '%s'" % lang['code']
+            else:
+                print "ERROR: fetching lang '%s'" % lang['code']
+    if tree_data:
+        tree = gh_repo.create_tree(tree_data, tree_sha)
+        message = 'OCA Transbot updated translations from Transifex'
+        c = gh_repo.create_commit(
+            message=message, tree=tree.sha, parents=[gh_branch.commit.sha],
+            author=gh_credentials, committer=gh_credentials)
+        gh_repo.ref('heads/{}'.format(gh_branch.name)).update(c.sha)
+
+
+def main():
+    projects = []
+    if args.projects:
+        # Check that provided projects are correct
+        for project_slug in args.projects:
+            try:
+                tx_project = tx_api.project(project_slug).get()
+                projects.append(tx_project)
+            except:
+                print "ERROR: Transifex project slug %s is invalid" % (
+                    project_slug)
+                return
+    else:
+        start = 1
+        temp_projects = []
+        print "Getting Transifex projects..."
+        while temp_projects or start == 1:
+            temp_projects = tx_api.projects().get(start=start)
+            start += len(temp_projects)
+            projects += temp_projects
+    for project in projects:
+        if 'OCA-' in project['slug']:
```

Comment:
Don't use hardcode "OCA"
Use a default value in config file
This will allow to use this script of universal way.